packages/@aws-cdk/aws-dynamodb-global/lib/aws-dynamodb-global.ts

```diff
@@ -0,0 +1,64 @@
+import dynamodb = require("@aws-cdk/aws-dynamodb");
+import cdk = require("@aws-cdk/cdk");
+import { LambdaGlobalDynamoDBMaker } from "./lambda-global-dynamodb";
+
+/**
+ * Properties for the mutliple DynamoDB tables to mash together into a
+ * global table
+ */
+export interface RegionalTablesProps extends cdk.StackProps, dynamodb.TableOptions {
+    /**
+     * Enforces a particular table name to share across Global DynamoDB tables
+     */
+    tableName: string;
+
+    /**
+     * Array of environments to create DynamoDB tables in
+     * The tables will all be created in the same account
+     */
+    regions: string[];
+}
+
+/**
+ * Global Tables builds upon DynamoDB’s global footprint to provide you with a fully managed, multi-region,
+ * and multi-master database that provides fast, local, read and write performance for massively scaled, global applications.
+ * Global Tables replicates your Amazon DynamoDB tables automatically across your choice of AWS regions.
+ * This class works by deploying an AWS DynamoDB table into each region specified in  GlobalTableProps.regions[],
+ * then triggering a CloudFormation Custom Resource Lambda to link them all together.
+ */
+export class RegionalTables extends cdk.Construct {
+    /**
+     * Creates the cloudformation custom resource that launches a lambda to tie it all together
+     */
+    private lambdaGlobalDynamodbMaker: LambdaGlobalDynamoDBMaker;
+
+    /**
+     * Creates dynamoDB tables across regions that will be able to be globbed together into a global table
+     */
+    private readonly _regionalTables = new Array<dynamodb.Table>();
+
+    constructor(scope: cdk.Construct, id: string, props: RegionalTablesProps) {
+        super(scope, id);
+        this._regionalTables = [];
+        if ( props.streamSpecification != null && props.streamSpecification !== dynamodb.StreamViewType.NewAndOldImages) {
+            throw(new TypeError("dynamoProps.streamSpecification MUST be set to dynamodb.StreamViewType.NewAndOldImages"));
+        }
+        // Need to set this streamSpecification here, otherwise global tables don't work
+        // And no way to set a default value in an interface
+        props.streamSpecification = dynamodb.StreamViewType.NewAndOldImages;
```

Comment:
If you rebase from master you'll see that all props are now readonly, so this will need to be rewritten like:

```ts
props = { ...props, streamSpecification: ... }
```